certbot/docs/using.rst

```diff
@@ -695,10 +695,47 @@ is done by means of a scheduled task which runs ``certbot renew`` periodically.
 
 If you are unsure whether you need to configure automated renewal:
 
-1. Review the instructions for your system at https://certbot.eff.org/instructions.
-   They will describe how to set up a scheduled task, if necessary.
-2. (Linux/BSD): Check your system's crontab (typically `/etc/crontab` and
-   `/etc/cron.*/*`) and systemd timers (``systemctl list-timers``).
+1. Review the instructions for your system and installation method at
+   https://certbot.eff.org/instructions. They will describe how to set up a scheduled task,
+   if necessary. If no step is listed, your system comes with automated renewal pre-installed,
+   and you should not need to take any additional actions.
+2. On Linux and BSD, you can check to see if your installation method has pre-installed a timer
+   for you. To do so, look for the ``certbot renew`` command in either your system's crontab
+   (typically `/etc/crontab` or `/etc/cron.*/*`) or systemd timers (``systemctl list-timers``).
+3. If you're still not sure, you can configure automated renewal manually by following the steps
+   in the next section. Certbot has been carefully engineered to handle the case where both manual
+   automated renewal and pre-installed automated renewal are set up.
+
+Setting up automated renewal
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you think you may need to set up automated renewal, follow these instructions to set up a
+scheduled task to automatically renew your certificates in the background. On the off chance you
+ended up at these instructions but automated renewal was pre-installed on your system, following
+these instructions should not cause any problems.
+
+If you're using Windows, these instructions will not work.
+
+Run the following line, which will add a cron job to the default crontab:
+
+.. code-block:: shell
+
+  SLEEPTIME=$(awk 'BEGIN{srand(); print int(rand()*(3600+1))}'); echo "0 0,12 * * * root sleep $SLEEPTIME &amp;&amp; certbot renew -q" | sudo tee -a /etc/crontab > /dev/null
+
+If you needed to stop your webserver to run Certbot, you'll want to
+add ``--pre-hook`` and ``--post-hook`` flags after ``certbot renew`` to stop
+and start your webserver automatically. For example, if your webserver is HAProxy, modify the
+command as follows:
```

Comment:
I have misgivings about this advice, though this probably isn't the PR to discuss this.

 For example, see [this forum thread from today](https://community.letsencrypt.org/t/renewing-certificats-and-configuring-on-windows-10/152593?u=_az). On any OS where preconfigured renewal is enabled, the user is not going to be able to make this adjustment.

If we're going to give out advice on how to stop/start a server using hooks, it should be catering to both preconfigured and manually-configured audiences:

- Doing it as part of lineage config (pass flags or modify renewal parameters), or
- Use the provided `hooks/{pre,post}` directories.

Reply:
It sure is copy-pasted. I do like having unified instructions if possible though, so we can discuss it here.

If we do it the lineage config way, this is not where those instructions go (they should go where we tell users how to run certbot, not set up renewal). I do worry that people might see this who didn't come from the instruction generator. In that case, what would we want here? It might in fact still be this, since it's the easiest and (I'm pretty sure) most common way to set things up, even if it's not the best.

I do actually think the directories are fine, since presumably they ran the initial issuance manually. And then we could keep it at this point in the instructions, which is appealing. It does mean it's no longer a one-line command, but I think that's fine.

Having written this all out, I've convinced myself to go towards the latter option, so I'll draft the new directions and have that up in a sec.